governance/.env.copy

```diff
@@ -9,4 +9,4 @@ export BSCSCAN_API_KEY=
 export SNOWTRACE_API_KEY=
 export ARBISCAN_API_KEY=
 export CELO_API_KEY=
-export GNOSISSCAN_API_KEY=
\ No newline at end of file
+export GNOSISSCAN_API_KEY=
```

Comment:
Why is this in this PR?

Reply:
I didn't add that, it was already in the .env.copy file

Reply:
its a deleted newline it seems

Reply:
yeah that change should *not* be in this PR.